test/jdk/java/awt/datatransfer/DataFlavor/DataFlavorRemoteTest.java

```diff
@@ -56,7 +56,7 @@ public static void main(String[] args) throws Exception {
         ProcessResults processResults =
                 ProcessCommunicator
                         .executeChildProcess(Consumer.class, new String[0]);
-        if (!"Hello".equals(processResults.getStdErr())) {
+        if (!"Hello".equals(processResults.getStdOut())) {
```

Comment:
Who or what prompted this change ?

Reply:
The child process is started with `-Djava.security.manager=allow` (after the other PR) and a warning will be printed to stderr. Therefore I move the message to stdout.